projects/amdsmi/amdsmi_cli/amdsmi_parser.py

```diff
@@ -300,15 +300,19 @@ def _power_cap_options(self):
         class AMDSMIPowerCapArgs(argparse.Action):
             def __call__(self, parser: AMDSMIParser, namespace: argparse.Namespace,
                          values: list, option_string: Optional[str] = None) -> None:
-                if len(values) != 2:
+                if len(values) == 1:
```

Comment:
I think this could still allow someone to give the power cap type and not the watts instead. This would then cause an exception down the line since we'd be expecting an integer value but getting a string in this case. I think if you change the if condition to like so:


```suggestion
                if len(values) == 1 and isdigit(values[0]):
```

Then that should cover us there.

Reply:
We already have a check for this in the else part:
<img width="502" height="218" alt="image" src="https://github.com/user-attachments/assets/8be20556-8479-4dbc-bad6-0ea57cb2ebad" />